### ISSUE TITLE
Change Mail Server

### ISSUE THREAD

**Issue by lvschnider** [grokability/snipe-it#14271]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I changed the mail server on my self hosted Snipe-IT but it still shows the default.
Mail From Address:Snipe-IT <you@example.com>Mail Reply-To Address:Snipe-IT <you@example.com>

### Reproduction steps

1.Changed .env to our mailserver
2.Ran artisan clear config and cache
3.Restarted apache
...

### Expected behavior

Change to our own mail server.

### Snipe-IT Version

v6.3.0 build 12490 (g9136415bb)

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

8.1.2-1ubuntu2.14

### Operating System

Mac

### Browser

Chrome

**Comment by snipe** [comment]
You must be editing the wrong file. If you've cleared the config using `php artisan config:clear`, the new changes should be accepted.

**Comment by lvschnider** [comment]
> You must be editing the wrong file. If you've cleared the config using `php artisan config:clear`, the new changes should be accepted.

I'm editing the .env file that is directly under the Snipe folder. Should I be editing a different one?

**Comment by snipe** [comment]
No, unless you have multiple Snipe-ITs installed,  but I can’t think of any reason clearing that config cache wouldn’t update your settings

**Comment by lvschnider** [comment]
I do not. Is there a way to see which .env file the instance is using?

**Comment by snipe** [comment]
It would have to be looking at the one within the snipe-it project directory that the files you're serving are in.

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
